src/plugin/utils/tests/StringTest.cpp

```diff
@@ -0,0 +1,64 @@
+#include <gtest/gtest.h>
+
+#include "String.hpp"
+
+using namespace ES::Plugin::Utils;
+
+TEST(String, Split)
+{
+    { // One split

```

Comment:
It count as multiple tests or just one ?

Reply:
Multiple tests, I'll split it into multiple `TEST` call

Reply:
Good to know thanks

Reply:
Done